src/lib/api-client.js

```diff
@@ -115,3 +115,11 @@ export function searchStudent(id){
         Dispatcher.handleAction('NO_STUDENT_FOUND');
     });
 }
+
+export function getAllModels(){
+    get('model/all', {}
+    ).then(data => {
+        Dispatcher.handleAction('MODELS_RECEIVED', data);
+        hashHistory.push('/models');
```

Comment:
Trevin will correct me if I'm wrong, but I believe this call should be made in the action itself in the store, not here in the api.